presto-memory-context/src/main/java/com/facebook/presto/memory/MemoryTrackingContext.java

```diff
@@ -107,9 +107,9 @@ public void close()
             closer.register(userAggregateMemoryContext::close);
             closer.register(revocableAggregateMemoryContext::close);
             closer.register(systemAggregateMemoryContext::close);
-            userLocalMemoryContext.setBytes(0);
-            revocableLocalMemoryContext.setBytes(0);
-            systemLocalMemoryContext.setBytes(0);
+            userLocalMemoryContext.close();
```

Comment:
Please search for other occurrences of `setBytes(0)`. It appears like there are other places where this is used.